django/core/management/templates.py

```diff
@@ -277,16 +277,22 @@ def cleanup_url(url):
 
         if self.verbosity >= 2:
             self.stdout.write('Downloading %s' % display_url)
+
         try:
-            the_path, info = urlretrieve(url, os.path.join(tempdir, filename))
+            response = urlopen(Request(
```

Comment:
What do you think about using an [opener](https://docs.python.org/3/library/urllib.request.html#urllib.request.build_opener) instead? For example:
```python
        the_path = os.path.join(tempdir, filename)
        opener = build_opener()
        opener.addheaders = [('User-agent', 'Django/%s' % get_version())]
        try:
            with opener.open(url) as source, open(the_path, 'wb') as target:
                headers = source.info()
                target.write(source.read())
        except OSError as e:
            ...
```

Reply:
That's very clean and neat indeed - thanks for the suggestion; applying.